model/telemetry/registry.yaml

```diff
@@ -81,3 +81,42 @@ groups:
         brief: >
           The version string of the auto instrumentation agent or distribution, if used.
         examples: ["1.2.3"]
+      - id: telemetry.sdk.processor.type
+        type: string
+        stability: experimental
+        brief: >
+          A name identifying the type of the OpenTelemetry SDK processor.
+        note: |
+          The value `batch-span` MUST be used for the SDK builtin Batch Span Processor.
+        examples: ["batch-span", "MyCustomProcessor"]
+      - id: telemetry.sdk.exporter.type
```

Comment:
same comment here, regarding enum type, since you#re proposing multiple well-known values to be used

Reply:
Fixed in 34e56e7